BotModules/SteamBot/index.js

```diff
@@ -31,13 +32,14 @@ module.exports = function(bot) {
   function PickRandomSharedGame(msg){
     let mentions = msg.mentions.users.map(user => user.id)
     if(mentions.length > 1){
-      MongoDB.GetCommonGames(mentions).then(games =>{
+      MongoConn = new MongoConnection()
+      MongoConn.GetCommonGames(mentions).then(games =>{
         let rdmIdx = Math.floor(Math.random() * games.length)
         msg.channel.send(`Lets play ${games[rdmIdx].name}!`)
-        MongoDB.Close()
+        MongoConn.CloseConnection()
       })
       .catch(err => {
-        MongoDB.Close()
+        MongoConn.CloseConnection()
         console.log(err)
       })
```

Comment:
Does Javascript have a finally clause? we could put the close connection there instead of writing it twice. Totally not a blocker though.

Reply:
It does, the finally clause for a promise looks a lot like weird, but works

ef73a7e